js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -67,48 +69,46 @@ const EmptyDashboard = () => {
   );
 };
 
+const FeatureItem = ({ feature }: { feature: FeatureRepresentation }) => {
+  const { t } = useTranslation("dashboard");
+  return (
+    <ListItem key={feature.name} className="pf-u-mb-sm">
+      {feature.name}{" "}
```

Comment:
I assume we want things to on the same line, so we could also use a non breaking space here.
```suggestion
      {feature.name}&nbsp;
```